packages/shared/src/components/squads/settings/SquadDangerZone.tsx

```diff
@@ -38,8 +38,8 @@ export function SquadDangerZone({ squad }: SquadDangerZoneProps): ReactElement {
       <DangerZone
         onClick={onDeleteSquad}
         className="mt-4"
-        cta="Delete account"
-        title="Deleting your account will:"
+        cta="Delete squad"
```

Comment:
Squad is a product term it's always capital S

```suggestion
        cta="Delete Squad"
```

Reply:
😬